src/components/Header.tsx

```diff
@@ -88,12 +89,18 @@ const Header = (): React.ReactElement => {
               target="_blank"
               variant="primary"
             />
+            <Navigation.Item
+              as="a"
+              href={`${config.ui.profileUIUrl}/loginsso`}
+              label="Helsinki-profiili"
+              target="_blank"
+            />
```

Comment:
I already approved this PR, but then I got an idea that how about the already existing link just above this, which now links to https://hel.fi, would act as the link to Profile UI?

Reply:
The ticket says the link's text should match the implementation in Jässäri. Consistency among all HKI services, I guess.

The nav item with user name and link to 'hel.fi' can be removed. User's name will remain in the Header anyway as dropdown's label, so nothing will be missed.

I'll create a new ticket for this cleanup.

Reply:
Good stuff.